js/components/Talent.js

```diff
@@ -8,11 +8,15 @@ class BudgetTalent extends HTMLElement {
     const talentContainer = document.createElement('div');
     const id = this.id;
     const name = this.name;
-    const budget = this.budget;
+    const expenditures = JSON.parse(this.expenditures);
     const talentTeamID = this.talentTeamID;
     const talentTeamName = this.talentTeamName;
-    //const expenditures = 
-    //const teams = 
+    
+    let budget = parseInt(this.budget);
+    for (let i = 0; i < expenditures.length; i++) {
+      budget = budget - expenditures[i].cost;     
```

Comment:
You are reassigning budget here. This is only possible because you changed it from a constant variable to a non-constant variable in line 15. 
I suppose you can just use `let budget`